### PR TITLE
Fix proxy issue when signing int to OpenShift Online Trials server.

### PR DESCRIPTION
**Pull request by odra**
Will not use a request proxy unless it is set on FHC config (fhc fhcfg).

[RHMAP-6397](https://issues.jboss.org/browse/RHMAP-6397)

### CODE REVIEW

File: lib/cmd/fhc/login.js [hunk 1]
```diff
@@ -167,4 +168,14 @@ function saveLogin(username, cookie, domain, targetUrl, cb) {
       return cb(err);
     });
   });
-}
\ No newline at end of file
+}
+
+function configureProxy() {
+  var proxy = fhc.config.get("proxy");
+
```
Comment (fh-qube) [inline lib/cmd/fhc/login.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by fh-automatron** [comment]
### Pull Request Quality Guidelines

As we continue to focus on quality, here are a few things to keep in mind:

* The code changes conform to the contributing & code style guides for this component (if they exist)
* A component owner should review your change (we've tagged these above)
* Automated testing has been added or updated for this change (e.g. UART, Unit/Acceptance tests)
* There should be an associated fhcap PR with this change - please provide a link to this
* If updating server component this PR should be deployed to a grid somewhere (via `fhcap-cli`) - please provide a link to this
* Ensure JIRA is up to date before & after merge (e.g. deployment notes, tester notes, release notes, fix version)
* Any SonarQube issues should be addressed where applicable (use your own judgement)

Please ping a component owner (see `fh contrib`) when your builds are complete, changes are deployed and you're ready for a review.

In order to update fhcap with your component change please execute:
`fhcap component update fh-fhc 2.7.3 302`

File: lib/utils/proxy.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+exports.configureProxy = function(req, proxy) {
```
Comment (fh-qube) [inline lib/utils/proxy.js]: BLOCKER Add a semicolon at the end of this statement. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)

**Comment by fh-qube** [comment]
SonarQube analysis reported 1 issue:
* BLOCKER 1 blocker

Watch the comments in this conversation to review them.

**Comment by pb82** [comment]
LGTM

File: package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "fhc": "./bin/fhc.js"
   },
   "publishConfig": {
-    "tag": "latest-2"
+    "tag": "latest-3"
```
Comment (wei-lee) [inline package.json]: I think the tag should not be changed.

**Pull request merged**